docs/source/usage.rst

```diff
@@ -122,26 +122,27 @@ Peeling arguments
                             for each metafounder after each peeling cycle.
       -no_phase_founder     A flag phase a heterozygous allele in one of the
                             founders (if such an allele can be found).
-      -sex_chrom            A flag to indicate that input data is for a sex chromosome. Sex needs to
-                            be given in the pedigree file. This is currently an
-                            experimental option.
-
+      -sex_chrom            A flag to indicate that input data is for a sex chromosome.
```

Comment:
Let's move to `x_chrom` as mentioned in https://github.com/AlphaGenes/AlphaPeel/issues/184